modules/metastorage/src/testFixtures/java/org/apache/ignite/internal/metastorage/impl/StandaloneMetaStorageManager.java

```diff
@@ -0,0 +1,135 @@
+import org.apache.ignite.internal.raft.service.CommandClosure;
+import org.apache.ignite.internal.raft.service.RaftGroupService;
+import org.apache.ignite.internal.util.IgniteSpinBusyLock;
+import org.apache.ignite.internal.vault.VaultManager;
+import org.apache.ignite.network.ClusterNode;
+import org.apache.ignite.network.ClusterService;
+import org.apache.ignite.network.NetworkAddress;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.TestOnly;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+
+/**
+ * MetaStorageManager dummy implementation.
+ */
+@TestOnly
+public class StandaloneMetaStorageManager extends MetaStorageManagerImpl {
+    private static final String TEST_NODE_NAME = "test";
+
+    /**
+     * Creates standalone MetaStorage manager for provided VaultManager.
+     */
+    public static StandaloneMetaStorageManager create(VaultManager vaultMgr) {
+        return create(vaultMgr, new SimpleInMemoryKeyValueStorage(TEST_NODE_NAME));
+    }
+
+    /**
+     * Creates standalone MetaStorage manager for provided VaultManager.
+     */
+    public static StandaloneMetaStorageManager create(VaultManager vaultMgr, KeyValueStorage storage) {
+        return new StandaloneMetaStorageManager(
+                vaultMgr,
+                Mockito.mock(ClusterService.class),
+                Mockito.mock(ClusterManagementGroupManager.class),
+                Mockito.mock(LogicalTopologyService.class),
+                Mockito.mock(RaftManager.class),
+                storage
+        );
+    }
+
+    private final KeyValueStorage keyValueStorage;
+
+    /**
+     * The constructor.
+     *
+     * @param vaultMgr Vault manager.
+     * @param clusterService Cluster network service.
+     * @param cmgMgr Cluster management service Manager.
+     * @param logicalTopologyService Logical topology service.
+     * @param raftMgr Raft manager.
+     * @param storage Storage. This component owns this resource and will manage its lifecycle.
+     */
+    public StandaloneMetaStorageManager(VaultManager vaultMgr, ClusterService clusterService, ClusterManagementGroupManager cmgMgr,
+            LogicalTopologyService logicalTopologyService, RaftManager raftMgr, KeyValueStorage storage) {
+        super(vaultMgr, clusterService, cmgMgr, logicalTopologyService, raftMgr, storage);
+
+        keyValueStorage = storage;
+    }
+
+    @Override
+    public void start() {
+        keyValueStorage.start();
+
+        MetaStorageListener metaStorageListener = new MetaStorageListener(keyValueStorage);
+
+        RaftGroupService raftGroupService = Mockito.mock(RaftGroupService.class);
+        Mockito.when(raftGroupService.run(ArgumentMatchers.any()))
+                .thenAnswer(invocation -> runCommand(invocation.getArgument(0), metaStorageListener));
+
+        var localNode = new ClusterNode(TEST_NODE_NAME, TEST_NODE_NAME, new NetworkAddress("localhost", 10000));
```

Comment:
`localNode` should be a constructor parameter

Reply:
Why? We pass fake ClusterNode just to avoid NPE. 
Standalone manager doens't imply any actual network interactions.

Reply:
oh, ok, let's leave it then